### PR TITLE
Disable zero degree check for commit.

### PR DESCRIPTION
**Pull request by 0xfluxx**

### CODE REVIEW

**Review by aterentic-ethernal — APPROVED** [review]

File: src/commitment_scheme/kzg10/key.rs [hunk 1]
```diff
@@ -161,10 +161,10 @@ impl CommitKey {
         &self,
         poly_degree: usize,
     ) -> Result<(), Error> {
-        match (poly_degree == 0, poly_degree > self.max_degree()) {
```
Comment (MiguelDD1) [inline src/commitment_scheme/kzg10/key.rs:164]: We should rename the function and update the documentation

Comment (0xfluxx) [reply src/commitment_scheme/kzg10/key.rs:164]: aggree! I'll do it asap!

Comment (0xfluxx) [reply src/commitment_scheme/kzg10/key.rs:164]: done

**Comment by prabal-banerjee** [comment]
Can we add a test case which checks commitment building and opening for the `0` degree case?

**Comment by 0xfluxx** [comment]
> Can we add a test case which checks commitment building and opening for the `0` degree case?

I have that test case in avail-core, adding it here might be a bit more time-consuming, but it's doable.

**Comment by 0xfluxx** [comment]
> Can we add a test case which checks commitment building and opening for the `0` degree case?

Actually, it was quite easy, I've added the two relevant cases.

**Pull request merged**